src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -5,8 +5,8 @@
 
 # TODO Move this to a package shared by CLI and SDK
 from functools import total_ordering
+from importlib import import_module
 from enum import Enum
-from importlib import import_module  # pylint: disable=wrong-import-order
```

Comment:
This is going to cause a wrong-import-order pylint error on whichever version of Python you aren't using...